<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelemetryConsole</class>
 <widget class="QMainWindow" name="TelemetryConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Constellation Telemetry Console</string>
  </property>
  <property name="windowIcon">
   <iconset resource="TelemetryConsole.qrc">
    <normaloff>:/logo_telemetry.png</normaloff>:/logo_telemetry.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="margin" stdset="0">
     <number>9</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0" columnstretch="0,6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>15</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="header" columnstretch="1,6,2,0" columnminimumwidth="0,0,0,0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>15</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="logo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="TelemetryConsole.qrc">:/logo_telemetry.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>71</height>
           </size>
          </property>
          <property name="title">
           <string>Constellation</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QLabel" name="cnstlnName">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>2</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&lt;font color='gray'&gt;&lt;b&gt;groupname&lt;/b&gt;&lt;/font&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="satelliteBox">
          <property name="title">
           <string>Senders</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="labelNrSatellites">
             <property name="text">
              <string>&lt;font color='gray'&gt;&lt;b&gt;0&lt;/b&gt;&lt;/font&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1258</width>
           <height>593</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,2">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBoxMetrics">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>63</height>
           </size>
          </property>
          <property name="title">
           <string>Add Metric</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1,3,1,2,1">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <widget class="QLabel" name="labelSender">
             <property name="text">
              <string>Sender</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="metricSender">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelName">
             <property name="text">
              <string>Metric</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="metricName"/>
           </item>
           <item>
            <widget class="QLabel" name="labelType">
             <property name="text">
              <string>Plot Type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="metricType">
             <item>
              <property name="text">
               <string>Spline</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Scatter</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Area</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addMetric">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Metrics</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="resetMetrics">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearMetrics">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="TelemetryConsole.qrc"/>
 </resources>
 <connections/>
</ui>
